<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tính Tiền Nước</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Số nước mới :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Tổng tiền :</string>
    </property>
   </widget>
   <widget class="QLabel" name="hinhAnhDaChon">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../../dvmv2/OneDrive/Pictures/Wallpaper/1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Địa chỉ :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Số điện thoại :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTittle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Tính Tiền Nước</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>Chụp ảnh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Họ tên :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Số nước cũ :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Tổng tiêu thụ :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Mã KH :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Mã Đồng Hồ :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChoose">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>Chọn Ảnh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaKH">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHoTen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDiaChi">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoDienThoai">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaDongHoNuoc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoNuocTruoc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoNuocSau">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTongTieuThu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTongTien">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Khối nước (m^3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>VNĐ</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1201</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 38, 52);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>451</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 43, 59);
 border: 2px solid rgb(1, 52, 88);      /* Blue outline */
    border-radius: 15px;            /* Rounded corners */</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>551</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 43, 59);
 border: 2px solid rgb(1, 52, 88);      /* Blue outline */
    border-radius: 15px;            /* Rounded corners */</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnSubmit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
      </property>
      <property name="text">
       <string>Xác Nhận</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
      </property>
      <property name="text">
       <string>Trở Về</string>
      </property>
     </widget>
     <widget class="QLabel" name="croppedImageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>531</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_6</zorder>
   <zorder>label_9</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lbTittle</zorder>
   <zorder>btnCapture</zorder>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_5</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>btnChoose</zorder>
   <zorder>txtMaKH</zorder>
   <zorder>txtHoTen</zorder>
   <zorder>txtDiaChi</zorder>
   <zorder>txtEmail</zorder>
   <zorder>txtSoDienThoai</zorder>
   <zorder>txtMaDongHoNuoc</zorder>
   <zorder>txtSoNuocTruoc</zorder>
   <zorder>txtSoNuocSau</zorder>
   <zorder>txtTongTieuThu</zorder>
   <zorder>txtTongTien</zorder>
   <zorder>label</zorder>
   <zorder>label_12</zorder>
   <zorder>hinhAnhDaChon</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
